### DISCUSSION TITLE
'IndexError: list index out of range' when saving reloaded data

### DISCUSSION THREAD

**Discussion by tom221085** [miniscope/minian#167] [Q&A] [answered]
Hello,

I posted already something about saving the data in the pipeline but now I'm having a similar issue. After I run the pipeline and saved the data, I sometimes want to reload it and run the cnmfviewer again to inspect ROIs. I use the following code to load the data:

![Capture1](https://user-images.githubusercontent.com/11939642/142504852-b8f4d72e-5a40-44cb-91a6-3d381140edf7.PNG)

After that I run the cnmfviewer but when I try to save the data again I get the following error message:

![Capture](https://user-images.githubusercontent.com/11939642/142504935-9035ce2e-dc61-4517-b71b-9200e6b50644.PNG)

Does anybody know what I am doing wrong here?

Thank you,
Thomas

**Comment by phildong** [comment, marked as answer]
tracking at #168
